### PR TITLE
tighten computer filesize regex

### LINKED ISSUE #153: If LDAP quota can't be parsed correctly, it should be used the ownCloud's default quota.
**Steps to reproduce:**

1. As admin set up an LDAP config with no value set in Quota Default.
2. Log in with an LDAP user  and check that the quota is the oC Default quota
3. As admin set Quota Default field with the value "@#00p"
4. Log in with an LDAP user  and check the quota.

**Expected behavior**

The quota should be oC Default quota or the previous value set

**Current behavior**

The quota is set to "0 B de 0 B (0 %)"

```
{"reqId":"rC360uIqkapATgCuWwvF","level":0,"time":"2017-11-27T11:13:09+00:00","remoteAddr":"85.57.168.79","user":"aaliyah_beer_6457","app":"user_ldap","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"using quota <@#00p> for user <aaliyah_beer_6457>"}
{"reqId":"57ZlyMxDgfT6A6pd4pJ1","level":0,"time":"2017-11-27T11:18:08+00:00","remoteAddr":"85.57.168.79","user":"aaliyah_beer_6457","app":"user_ldap","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"No LDAP quota attribute configured"}
```

**LDAP app version:** 

release/0.10.0

### PR DESCRIPTION
**Pull request by butonic**
fixes https://github.com/owncloud/user_ldap/issues/153

### CODE REVIEW

File: core/js/js.js [hunk 1]
```diff
@@ -1750,7 +1750,7 @@ OC.Util = {
 	* regular expression to parse size in bytes from a humanly readable string
 	* see computerFileSize(string)
 	*/
-	_computerFileSizeRegexp: /^[\s+]?([0-9]*)(\.([0-9]+))?( +)?([kmgtp]?b?)$/i,
+	_computerFileSizeRegexp: /^([0-9]*)(\.([0-9]+))?( +)?([kmgtp]?b?)$/i,
```
Comment (butonic) [inline core/js/js.js]: string is trimmed before matching anyway

**Comment by davitol** [comment]
Tested and solves https://github.com/owncloud/user_ldap/issues/153 but seems there are some automated tests failing

**Comment by PVince81** [comment]
we'll anyway need to adjust tests that cover the changes made here

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
@butonic please backport

**Pull request merged**